catalogs/usfimr-s1/build_catalog.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+from datetime import datetime
+import json
+import argparse
+from itertools import groupby
+
+from pystac import (
+    Asset,
+    CatalogType,
+    Catalog,
+    Collection,
+    Extent,
+    Item,
+    SpatialExtent,
+    TemporalExtent,
+)
+import boto3
+import urllib3
+import rasterio as rio
+from shapely.geometry import Polygon, mapping
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--imagery-root-s3", required=True)
+    args = parser.parse_args()
+
+    parsed_s3_path = urllib3.util.parse_url(args.imagery_root_s3)
+    bucket = parsed_s3_path.netloc
+    prefix = parsed_s3_path.path[1:]
```

Comment:
Minor nitpick but this line fails with `TypeError: 'NoneType' object is not subscriptable`

If my imagery root is just a bucket: `--imagery-root-s3 s3://glofimr-sar-4326`

I can work around it with `--imagery-root-s3 s3://glofimr-sar-4326/` (add a slash to the end) but would be nice to not throw a cryptic error.